### ISSUE TITLE
Envato Market Plugin Not Working? Again?

### ISSUE THREAD

**Issue by Evelyn7** [envato/wp-envato-market#55]
I installed the Envato Market Plugin and Activated it and then I put in my Key - it was accepted. I clicked the "update" for my Theme and it has been "updating" for over 8 hours! What is going on? Please advise!

**Comment by valendesigns** [comment]
You should never wait that long. If it doesn't work in a minute or two refresh the page and try again.

**Comment by Evelyn7** [comment]
Thank you, @valendesigns for your quick response!
Oh... I did as you suggested just now and instead of simply updating the theme (as I've done with this exact theme in the past via the old Envato Tookit Plugin) - it all vanished! And, now when I click on the theme to "install" it it simply goes to some landing page that I can only assume it the theme authors...? (Completely off of my site, away from my WordPress login entirely) I am quite upset over losing my entire site. What is going on?? Please advise.

**Comment by dtbaker** [comment]
Hey please let us know if this is still an issue and we can look into it further.

**Comment by Evelyn7** [comment]
Hi+
I downloaded the zip file from your website on 02/26/18, then removed the old plugin and installed the new zip file via the back-end Plugins > add new > upload. I then activated - and it still did not work. It did show an "update" button/link, however, when I clicked that nothing happened.

WP v 4.9.4
Envato Plugin v. 1.0.0-RC2

So, is the zip file on your website the "new" (fixed) version of the plugin? If so, why is it not working as expected? If not, where do I find the correct version?
Thanks!
